README.md

```diff
@@ -4,77 +4,126 @@
-```
+#### Goal 3: Give web app developers a standard wallet API
 
-If you plan to develop or modify this polyfill, install it from Github:
+Web app developers should not be required to roll their own wallet 
+infrastructure for every application. Using a standard credential management
+API allows developers to:
 
-```
-git clone https://github.com/digitalbazaar/credential-handler-polyfill.git
-cd credential-handler-polyfill
-npm install
-```
+* Implement secure user account and login systems
+* Minimize the risk of data leaks and identity theft
+* Participate in the [Verifiable Credentials](https://w3c.github.io/vc-data-model/) 
+  ecosystem  
+
+#### About this Polyfill
+
+TODO: Explain the importance of polyfills for driving innovation and standards
+for the Web platform. (a.k.a, many of your favorite Web platform features
+probably started out as a polyfill.) 
+
+#### Relationship to Other Standards
+
+TODO: Add section explaining how CHAPI interacts with existing complementary
+web standards:
+
+* WebAuthn and FIDO
+* Credential Management Level 1 API
+* Verifiable Credentials
+* DIDs
+
+## Demo
+
+Visit https://digitalbazaar.github.io/credential-handler-polyfill/demo/ 
+for an idea of what the basic handler UI looks like.
 
 ## Usage
 
 ### Loading the Polyfill
 
+Before you can get and store credentials or keys, you need to load the
+polyfill library.
+
+If you're [loading the polyfill from a `<script>` 
+tag](#including-credential-handler-api-in-browser-scripts), you will have
+access to the `navigator.credentials` and `credentialHandlerPolyfill` globals.
+
+```js
+const polyfill = window.credentialHandlerPolyfill;
+```
+
+Otherwise (if you're developing on Node.js and using Webpack, for example),
+import it in the usual manner:
+
 ```js
 import * as polyfill from 'credential-handler-polyfill';
+````
 
-const mediatorUrl = 'https://beta.authn.io/mediator?origin=' +
-  encodeURIComponent(window.location.origin);
+Load (async):
 
-await polyfill.loadOnce(mediatorUrl);
-console.log('loaded...')
+```js
+await polyfill.loadOnce();
+console.log('Ready to work with credentials!');
 ```
 
-### Installing a Credential Handler
+### Requesting and Storing Credentials
+
+A web application can `get()` and `store()` credentials without knowing anything
+about the user's wallet. This is intentional; for privacy reasons, the client
+app must not have access to any information regarding which wallets or 
+credential handlers a user may have installed (otherwise, fingerprinting and
+other attacks would be possible).
 
-Once the polyfill is loaded, you can install it:
+#### `get()`
+
+A web app (a Relying Party or **verifier**) can request a credential using
+`credentials.get()`, for example when the user 
+pushes a button on a page that requires identity attributes or authentication.
 
 ```js
-const {CredentialManager, CredentialHandlers} = navigator.credentialsPolyfill;
+const webCredential = await navigator.credentials.get({web: {}});
```

Comment:
Add TODO about updating `web: {}` to something more useful.

Reply:
Done, thanks.